sources/install/package_base.sh

```diff
@@ -173,6 +173,16 @@ function install_ultimate_vimrc() {
     sh ~/.vim_runtime/install_awesome_vimrc.sh
 }
 
+function install_neovim() {
+    curl -LO https://github.com/neovim/neovim/releases/latest/download/nvim.appimage
```

Comment:
This installation will not work with the arm architecture : 

```bash
> ./nvim.appimage --appimage-extract

zsh: exec format error: ./nvim.appimage
```

ARM install :

```bash
wget https://github.com/neovim/neovim/releases/download/stable/nvim-macos.tar.gz
```